File: Server/src/main/java/xyz/sleepylux/hextechfriends/models/DTO/client/RegisterLobbyTS.java
package xyz.sleepylux.hextechfriends.models.DTO.client;

public class RegisterLobbyTS {
    private String summonerName;
    private int level;
    private int iconId;
    private String ownerId;
    private String opCode;
    private String platform;

    public String getSummonerName(){ return this.summonerName; }
    public int getLevel(){ return this.level; }
    public int getIconId(){ return this.iconId; }
    public String getOwnerId(){ return this.ownerId; }
    public String getPlatform() { return this.platform; }
}

File: Client/HextechFriendsClient/Protocol/Client/RegisterLobbyTS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HextechFriendsClient.Protocol.Client
{
    class RegisterLobbyTS
    {
        public string summonerName { get; set; }
        public int level { get; set; }
        public int iconId { get; set; }
        public string ownerId { get; set; }
        public string platform { get; set; }
        public string opCode = "registerLobbyTS";
    }
}
